hack/config.sh

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+unset docker_tag master_ip network_provider kubeconfig manifest_docker_prefix namespace image_pull_policy
+
+KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-${PROVIDER}}
+
+source ${KUBEVIRT_PATH}hack/config-default.sh source ${KUBEVIRT_PATH}hack/config-${KUBEVIRT_PROVIDER}.sh
```

Comment:
is this a bug? i don't think you can call source twice like this

Reply:
works for kubevirt :)

https://github.com/kubevirt/kubevirt/blob/master/hack/config.sh#L6

Reply:
I need to see how it works, cannot believe it

Reply:
the 2nd "source" and the file following are actually $1 and $2 to the execution of "config-defaults.sh"
check what "config-defaults.sh" does with its arguments.
even if it works it is pretty ugly :-(

Reply:
@SchSeba does not work for me:

```shell
➜  test cat env1.sh 
A=1
B=2
➜  test cat env2.sh
A=11
➜  test source env1.sh source env2.sh 
➜  test echo $A
1
```

Reply:
ok I think this can be removed the file `${KUBEVIRT_PATH}hack/config-${KUBEVIRT_PROVIDER}.sh` never exist .

The file name is `hack/config-provider-${KUBEVIRT_PROVIDER}.sh` and we have a check for that on the next line.
`# Allow different providers to override default config values
test -f "hack/config-provider-${KUBEVIRT_PROVIDER}.sh" && source hack/config-provider-${KUBEVIRT_PROVIDER}.sh`